broker/models.py

```diff
@@ -1,6 +1,7 @@
 from enum import Enum
 from sqlalchemy.dialects import postgresql
 import sqlalchemy as sa
+from sqlalchemy.orm import mapped_column
```

Comment:
sqlalchemy 1.x -> 2.x change. 
`mapped_column` replaces `db.Column`, mostly as a drop-in